src/main/java/uk/ac/ebi/ena/webin/cli/manifest/ManifestReader.java

```diff
@@ -297,6 +321,87 @@ public void addListener(MessageListener listener) {
         return null;
     }
 
+    private void parseManifestNew( Path inputDir, List<String> lines ) {
+        state.state = PARSE;
+
+        try {
+            ObjectMapper objectMapper = new ObjectMapper();
+
+            JsonNode jsonNode = objectMapper.readTree(lines.stream().collect(Collectors.joining("\n")));
+
+            jsonNode.fields().forEachRemaining(field -> {
+                String fieldName = field.getKey();
+
+                if( fieldName.matches( FIELD_NAME_REGEX ) )
+                    return;
+
+                //find field definition
+                ManifestFieldDefinition fieldDefinition = Stream.concat( infoFields.stream(), fields.stream() )
+                        .filter(fieldDef -> fieldDef.getName().equalsIgnoreCase( fieldName ) ||

```

Comment:
Could we also please do punctuation insensitive field name matching (we could use this in keyValue field parsing as well and in attribute parsing):

```
public static boolean matchNameCaseAndPunctuationInsensitively(String name1, String name2) {
   return name1.replaceAll("<space><tab><underscore><dash/minus>").equalsIgnoreCase(name2.replaceAll("<space><tab><underscore><dash/minus>");
}
```
so that all of the following are ok:
- fastqFile
- fastq_File
- FASTQ-FILE